src/core_modules/capture-core/components/WidgetEventSchedule/CategoryOptions/CategoryOptions.component.tsx

```diff
@@ -66,22 +64,22 @@ const CategoryOptionsPlain = (props: Props) => {
         required,
     } = props;
 
-    const renderCategorySelector = (category) => {
+    const renderCategorySelector = (category: CategoryOption) => {
         const { id } = category;
         const hasError = categoryOptionsError?.[id]?.touched && !categoryOptionsError?.[id]?.valid;
         return (<div className={hasError ? classes.error : ''}>
             <div className={orientation === 'horizontal' ? classes.container : classes.containerVertical}>
-                <div className={orientation === 'horizontal' && classes.label}>
+                <div className={orientation === 'horizontal' ? classes.label : ''}>
                     {category.displayName}
                     {required && <span
                         className={classes.requiredClass}
                     >
                     &nbsp;*
                     </span>}
                 </div>
-                <div className={orientation === 'horizontal' && classes.field}>
+                <div className={orientation === 'horizontal' ? classes.field : ''}>
                     <CategorySelector
-                        initialValue={selectedCategories?.[category.id]}
+                        initialValue={selectedCategories?.[category.id] ? { label: '', value: selectedCategories[category.id] } : null}
```

Comment:
Rollback to `initialValue={selectedCategories?.[category.id]}` and fix the typescript error by using  `selectedCategories: {[categoryId: string]: any } | null;` on line 49